src/js/custom-javascript.js

```diff
@@ -1 +1,36 @@
+/**
```

Comment:
don't call it custom-javascript.js. just call it bootstrap.js and leave custom-javasctipt.js for custom javascript

Reply:
Good idea- I see I also wasn't bundling the skip-link-focus-fix.js file, so I fixed that. But now you can just modify `src/js/bootstrap.js` to remove any unnecessary bootstrap modules.